Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailsViewController.swift

```diff
@@ -209,7 +209,10 @@ class TransactionDetailsViewController: LoadableViewController, UITableViewDataS
         guard let nonce = safe.nonce, nonce == tx?.multisigInfo?.nonce.value else {
             return false
         }
-        return tx?.needsYourExecution ?? false
+        guard let tx = tx else {
+            return false
+        }
+        return needsYourExecution(tx: tx)
```

Comment:
Maybe this should be needsExecution? Since now we allow anyone to execute?

Reply:
I expect to refactor this later and would not spend more time on the changes in this pr :pray: